open_academy/models/open_academy_session.py

```diff
@@ -0,0 +1,14 @@
+from odoo import models, fields
+
+
+class OpenAcademySession(models.Model):
+    _name = "open.academy.session"
+    _description = "model of session on open academy"
+
+    name = fields.Char(required=True, size=50)
+    duration = fields.Text()
+    start_date = fields.Datetime()
+    number_seats = fields.Integer()
+    instructor_id = fields.Many2one("res.partner", "instructor")
+    course_id = fields.Many2one("open.academy.course", required=True)
+    attendes_ids = fields.Many2many("res.partner", "attendees")
```

Comment:
As a good practice we always place for the name of the relation, the name of both models and the suffix `_rel`. The first name is current and the second is related.
```suggestion
    attendees_ids = fields.Many2many("res.partner", "session_res_partner_rel")
```